api/src/main/java/gov/cms/ab2d/api/controller/StatusAPI.java

```diff
@@ -193,7 +193,7 @@ private JobCompletedResponse getJobCompletedResonse(Job job) {
     )
     @DeleteMapping(value = "/Job/{jobUuid}/$status")
     @ResponseStatus(value = HttpStatus.ACCEPTED)
-    public ResponseEntity<Void> deleteRequest(
+    public ResponseEntity deleteRequest(
```

Comment:
Was void actually causing an issue here as the generic type?

Reply:
It was. It was saying that Void was not defined in the model.